content/en/docs/contribute/localization.md

```diff
@@ -263,7 +270,7 @@ To collaborate on a development branch:
 
 Repeat steps 1-4 as needed until the localization is complete. For example, subsequent German development branches would be: `dev-1.12-de.2`, `dev-1.12-de.3`, etc.
 
-Teams must merge localized content into the same release branch from which the content was sourced. For example, a development branch sourced from {{< release-branch >}} must be based on {{< release-branch >}}.
+Teams must merge localized content into the same release branch from which the content was sourced. For example, a development branch sourced from `{{< release-branch >}}` must be merged into `{{< release-branch >}}`.
```

Comment:
```suggestion
{{< note >}}
Localization work should target the current release. If a minor (or major) Kubernetes release has happened since a localization branch was started, create a new development branch based on either `master` or on an ancestor of `master` _that includes updates for the current release_, and merge in any existing development branch that was working on documenting a previous release.
{{< /note >}}
```
seems more accurate / appropriate.

Reply:
Isn't it "a descendant", not "an ancestor"?
I'm drawing relationship between branches as follow and I'd like to add it into here, please take a look also.
https://hackmd.io/smrEVJCJRee3cvXTIQ7bJA

Reply:
Ancestor is correct; I'm describing the **merge base** of the new localization branch that you're creating.. If after you start, someone adds a commit to _master_ and not to your branch, that new commit to _master_ does not invalidate your work so far (although you might need to address conflicts).

Reply:
Some l10n teams working on previous releases, so `Localization work should target the current release.` is too strict, I think.
Also, could you simplify the long sentence in note? Long sentence tends to be difficult to understand. Thanks.

Reply:
I rephrased this section as in new commit. PTAL.